jupyterhub/handlers/base.py

```diff
@@ -994,7 +1002,7 @@ def _fail_api_request(self, user):
                 and spawner._spawn_future.exception()
             ):
                 # Condition: spawner not active and _spawn_future exists and contains an Exception
-                # Implicit spawn on /user/:name is not allowed if the user's last spawn failed.
+                # Implicit spawn on /user/:user_name is not allowed if the user's last spawn failed.
```

Comment:
leave `:name` here

Reply:
Do you want me to change all of the `user_name`'s back to `name`, or just this one?